packages/scripts/utils/changelog/requests.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const { REPO } = require( '../../config/changelog.config' );
+const { authedGraphql } = require( './get-entry' );
+
+/* eslint no-console: 0 */
+
+const getMilestoneNumber = async ( version ) => {
+	const [ owner, repo ] = REPO.split( '/' );
+	const query = `
+	{
+		repository(owner: "${ owner }", name: "${ repo }") {
+			milestones(last: 50) {
+				nodes {
+					title
+					number
+				}
+			}
+		}
+	}
+	`;
+	const data = await authedGraphql( query );
+	const matchingNode = data.repository.milestones.nodes.find(
+		( node ) => node.title === version
```

Comment:
I see that the Gutenberg project names milestones with `Gutenberg {version num}`.  So this should probably match on the title containing the provided version number.

Reply:
OR, I see from your example you provided the exact milestone title to match on, in which case probably you can change the question prompting during the script to be something like:

> In order to generate the changelog, you will have to provide a milestone title to retrieve the PRs from.